### ISSUE TITLE
[ChannelTabs] Issue: throws react error on context menu

### ISSUE THREAD

**Issue by TheAwesome98-Real** [l0c4lh057/BetterDiscordStuff#161]
**Describe the bug**
When a context menu that has ChannelTabs buttons in it is opened, a react error is thrown.

**To Reproduce**
1. Right click on anything with the plugin enabled

**Expected behavior**
No exception is thrown and the context menu is opened.

**Screenshots**
exception is minified so i don't think you'll need it?

**Information (please complete the following information):**
 - OS: [e.g. Windows, MacOS, TempleOS] Linux
 - Plugin Version: [e.g. 1.2.3] 2.6.5
 - Discord version: [PTB, Stable, Canary] Stable
 - Compact Mode enabled? [yes or no] No

**Comment by MichaelDavidGK** [comment]
I believe that I once got non-minified error that said something about that it cannot take "group" when "object" is expected, anyway it is due to recent updates ([1](https://github.com/BetterDiscord/BetterDiscord/commit/9d7264be96b48ce1c96cb4a8da55f81c80959fdf), [2](https://github.com/BetterDiscord/BetterDiscord/commit/5b8a173aa0a718f21b8403c84827de3df8d95818))

**Comment by Matt0040** [comment]
This issue happens with just about any plugin that affects the menu created when you right click a servers name, user name, anything to get some specific details like user ID. It took a long time but I was able to fix it, Channel Tabs obviously is one but others would be the BDContextMenu. The console reads something about the new API being unable to use children. If I weren't scared of fixing what's not broken I'd try to get a console log. 

TL;DR Discord crashes whenever a plugin affecting context menus right clicks a user, server, almost anything from what I can tell.

**Comment by TheAwesome98-Real** [comment]
oh, i forgot, i did update discord to 0.22.0 recently

**Comment by MichaelDavidGK** [comment]
It has been fixed previously and now version number for BD has been changed to 1.8.4 to ensure everyone gets the fix:
[FIXED](https://github.com/BetterDiscord/BetterDiscord/commit/daafbf51b135a9d903a26c79623e7b0d319b7be9)

As such this problem is fixed ;)

**Comment by TheAwesome98-Real** [comment]
fixed in betterdiscord 1.8.4
